modules/lvms-deleting-volume-snapshots.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-deleting-volume-snapshots_{context}"]
+= Deleting volume snapshots
+
+You can delete the volume snapshots of the persistent volume claims (PVCs).
+[IMPORTANT]
+====
+When you delete a persistent volume claim (PVC), {lvms} deletes only the PVC but not the snapshots of the PVC.
```

Comment:
Add comma after "PVC": "...persistent volume claim (PVC), {lvms} deletes only the PVC, but not the snapshots..."

Reply:
Updated as per the suggestion.